packages/legend-studio-extension-query-builder/src/components/QueryBuilder_StudioPlugin.tsx

```diff
@@ -226,6 +234,41 @@ export class QueryBuilder_StudioPlugin
                   element,
                 );
                 queryBuilderExtension.queryBuilderState.resetData();
+                if (element.stereotypes.length !== 0) {
+                  const parmaterState = new QueryParameterState(
+                    queryBuilderExtension.queryBuilderState.queryParametersState,
+                    new VariableExpression(
+                      'businessDate',
```

Comment:
We should bring this out to legend-query as a constant. Something like `DEFAULT_VERSION_PARAMETER_NAME`